swift-sdk/Internal/RequestCreator.swift

```diff
@@ -418,6 +420,23 @@ struct RequestCreator {
         return .success(.post(createPostRequest(path: Const.Path.trackInboxSession, body: body)))
     }
     
+    // MARK: - Embedded Messaging Request Calls
+    
+    func createGetEmbeddedMessagesRequest() -> Result<IterableRequest, IterableError> {
+        if case .none = auth.emailOrUserId {
+            ITBError(Self.authMissingMessage)
+            return .failure(IterableError.general(description: Self.authMissingMessage))
+        }
+        
+        var args: [AnyHashable: Any] = [:]
+        
+        setCurrentUser(inDict: &args)
```

Comment:
will need `userKey` and `placementId` in it as well.

Reply:
Yep, that's covered by `setCurrentUser(inDict: &args)`. I'll have `placementId` in another branch.